services/movie_session.py

```
@@ -0,0 +1,52 @@
+from datetime import datetime
+
+from django.db.models import QuerySet
+
+from db.models import MovieSession, Movie, CinemaHall
+
+
+def create_movie_session(
+        movie_show_time: datetime,
+        movie_id: int,
+        cinema_hall_id: int
+) -> MovieSession:
+    return MovieSession.objects.create(
+        show_time = movie_show_time,
+        movie_id = movie_id,
+        cinema_hall_id = cinema_hall_id
+    )
+
+
+def get_movies_sessions(session_date: str = None) -> QuerySet:
+    queryset = MovieSession.objects.all()
+    if session_date:
+        queryset = queryset.filter(show_time__date=session_date)#!!!
+
+    return queryset
+
+
+def get_movie_session_by_id(
+        movie_session_id: int
+) -> MovieSession:
+    return MovieSession.objects.get(id=movie_session_id)
```

The `get_movie_session_by_id` function uses `MovieSession.objects.get(id=movie_session_id)`, which will raise a `DoesNotExist` exception if the session ID is not found. Consider using `get_object_or_404` if this function is used in a view, or handle the exception appropriately.